### DISCUSSION TITLE
Using a CSV field as a filter on a condition isn't working in the admin app

### DISCUSSION THREAD

**Discussion by mark-james** [directus/directus#11302] [Archive]
Hi,
I'm trying to use a multi-select dropdown based on a CSV field to trigger the visibility of other fields within a collection. But it isn't working. I know JSON doesn't work due to postgres limitations but I believe CSV is supposed to?

The setup. A multi-select dropdown (Blocks field) with Description selected. The Description field is still visible.
![Screen Shot 2022-01-27 at 4 58 13 pm](https://user-images.githubusercontent.com/26987752/151301038-79ad7381-fe8b-44c2-8171-2e6f6ecee171.png)

The raw value is set to Description
![Screen Shot 2022-01-27 at 4 58 21 pm](https://user-images.githubusercontent.com/26987752/151301074-9b8ffae9-c784-4e93-b02c-bf3046062bde.png)

And the filter in the condition is shown as Description. The visibility checkbox is ticked.
![Screen Shot 2022-01-27 at 4 57 12 pm](https://user-images.githubusercontent.com/26987752/151301339-ac969563-a43a-4d15-91d1-a435705917a3.png)

Is this a bug or am I doing something wrong?
(I'm on the latest version, 9.5.0)

Thanks
Mark

**Comment by benhaynes** [comment]
Likely a bug. CSVs should work with the `contains` operator... and if not, then it should be disabled when working with the CSV type.

**Reply by mark-james** [reply]
Thanks Ben, issue raised #11309
